### ISSUE TITLE
Support for neo4j-gremlin 3.4.8

### ISSUE THREAD

**Issue by kuzeko** [neo4j-contrib/neo4j-tinkerpop-api-impl#21]
Is `neo4j-gremlin` `3.4.8` supported yet? I cannot find in maven central the corresponding dependency
latest is `0.9-3.4.0` ?

**Comment by jexp** [comment]
You should be able to just update the dependency locally.

**Comment by kuzeko** [comment]
You mean  I should change it in the pom  and compile it locally?

**Comment by jexp** [comment]
Yes.

I can actually not remember, if it bundles neo4j jars or just links to them. In the latter case you should be able to just replace them with the newer version.

**Comment by kuzeko** [comment]
What if I do that and provide a pull request?

**Comment by jexp** [comment]
sure, go ahead. Thank you.

You will need to sign our CLA though https://neo4j.com/developer/cla

**Comment by kuzeko** [comment]
So I got this error while bulding:

```
Could not find artifact org.neo4j:neo4j-enterprise:jar:3.5.21 in central (https://repo.maven.apache.org/maven2)
```

So this means I need to access those jars locally somehow I guess...?

**Comment by jexp** [comment]
Yes since 3.5 the enterprise jars are no longer in maven central just in the binary distribution.

Also as HA has been removed from 4.0 onwards, I would say the best thing is to remove the Cluster/Enterprise bits from the code.

Customers/users that have access can put the dependencies in if needed.

**Comment by kuzeko** [comment]
Ok, so I cannot compile it?
I did a test and put there the latest 3.5-beta for the enterprise and it compiled.

Here I am working on the 3.5 branch yet, so I think HA can stay there

**Comment by kuzeko** [comment]
Hi, I have this pull request pending, but not sure how to fix it

https://github.com/neo4j-contrib/neo4j-tinkerpop-api-impl/pull/22
